Automattic/wp-calypso
# Jetpack Plans: Try tabs layout

This PR enables an A/B test that shows a tabbed version of the plans page during the jetpack connection flow:

![image](https://cloud.githubusercontent.com/assets/1554855/22022770/a7071d06-dcc3-11e6-8df4-18bddf17da03.png)

How to test:
0. Go to http://calypso.localhost:3000/jetpack/connect/store and make sure you get the correct variation of the test opening the dev console and writting: 
`localStorage.setItem( 'ABTests', '{"jetpackPlansTabs_20170117":"tabs"}' )`

1. Go to this urls:
  http://calypso.localhost:3000/jetpack/connect/store
  http://calypso.localhost:3000/jetpack/connect/vaultpress
  http://calypso.localhost:3000/jetpack/connect/akismet 
  And make sure you get the tabbed version of the plans page. Play with the tabs and change the different settings and make sure everything works fine
2. Once you finish, connect a jetpack site. Make sure that you get the tabbed version of the plans page once the connection is complete.

ping @richardmuscat @rickybanister @roccotripaldi
